pkg/ui/admin/embed/static/js/namespaces.js

```diff
@@ -0,0 +1,56 @@
+function handleUpdateNamespace() {
+  $("#updateForm").on("submit", function(event) {
+    event.preventDefault(); // Prevent the default form submission
+
+    // Get form data
+    const formData = $(this).serializeArray();
+
+    // Convert formData to a regular object
+    const formDataObject = {};
+    formData.forEach(function(entry) {
+      formDataObject[entry.name] = entry.value;
+    });
+
+    // Perform a PUT request using jQuery's $.ajax
+    $.ajax({
+      url: "/admin/ns/" + formDataObject["id"],
+      type: "PUT",
+      contentType: "application/json",
+      data: JSON.stringify(formDataObject), // Convert to JSON format
+    }).done(handleResponse);
+  });
+}
+
+function createNamespace() {
+  redirectTo('/admin/ns/new');
+}
+
+function editNamespace(id) {
+  redirectTo('/admin/ns/' + id);
+}
+
+function namespaceIndex() {
+  redirectTo('/admin/ns/');
+}
+
+function redirectTo(path) {
+  window.location = window.location.origin + path;
+}
+
+function deleteNamespace(id) {
+  if (confirm("Are you sure?") != true ){
+    return
+  }
+  // Perform a DELETE request using jQuery's $.ajax
+  $.ajax({
```

Comment:
if we want to use REST methods we need to do this, but we could just post to named endpoints, too.

Reply:
Im not fully sure about `ajax`, because it looks a bit strange like:
- we do `ajax` call - ok.
- and then we just redirect - strange.
for me in case of `ajax` way it should look like:
- we do `ajax` call.
- we process response.
- we request updated list without page reloading.
- we show the list of namespaces.
so it seems like on current step we can simplify it even more and just send `form` data without `ajax`. @jgiannuzzi what do you think?

Reply:
we cannot use PUT, DELETE http methods without Ajax, so they'll need to instead have named endpoints, eg "/delete"